docs/platform/tutorials/currency-alerting.md

```diff
@@ -1,256 +1,242 @@
 
-1.  Create third party accounts: Twilio and CoinApi
+## Setting Up The Functions
+Now let’s deploy the functions we need to build the pipeline.  Luckily, we don’t have to write any of these functions from scratch. We can import them from the Quix library and adapt a few variables for this tutorial.
 
-2.  Deploy connectors from the Quix Library for:
-    
-    1.  the Coin API data source
-    
-    2.  the Twilio sink or output
 
-## Create free third party accounts
+!!! note "What is the Quix Library?"
 
-### Create a free CoinApi account
+	The Quix library is a repository of useful functions in Python (and other supported languages) categorized by each of the three roles. The library includes connector functions that ingest data or output data. It also includes transformation functions that can process and transform the data in different ways.
 
-!!! note
+Let’s set up each function one by one, starting with our input function.
 
-	CoinAPI is a platform that provides data APIs for many financial instruments including cryptocurrency. We will be using it to get live Bitcoin (BTC) data.
 
-Let’s create a free coinAPI account:
+### Setting Up The CoinAPI Input Function
 
-1.  Go to [https://www.coinapi.io/](https://www.coinapi.io/){target=_blank}.
+**Purpose:**  _Ingest BTC/USD Exchange Rate Data_
 
-2.  Click the "Get a free API key" button and complete the dialog to
-    create a free account.
+This function connects reads a live stream of updates for the currency pair: `BTC/USD`. We get this data by connecting to the [CoinAPI](https://www.coinapi.io/) [Websocket](https://en.wikipedia.org/wiki/WebSocket) (the free [sandbox version](https://docs.coinapi.io/#endpoints-2)). 
 
-3.  You will receive your API key on your provided email. **Keep it safe
-    for later**.
+* The function streams the exchange rate data to a topic called `currency-exchange-rates`. 
+* Downstream functions can then read fom this topic and process it in different ways. 
+* In this case, we're going to check the price against a threshold.
 
-### Create a free Twilio account
 
-!!! note
+**To set up the CoinAPI input function, follow these steps:**
 
-	Twilio is a developer platform for communications. We can use Twilio, for instance, to send SMS messages.
+1. In the [Quix Portal](https://portal.platform.quix.ai/), open the Quix Library<br>![Library icon](../images/tutorials/currency-alerting/library-icon.png)
+   <br><br>
+2. In the search box on the library page, search for "_CoinAPI—Exchange Rate Feed_".<br>
+   You should see a result that resembles the following example: ![CoinAPI library item](../images/tutorials/currency-alerting/coinapi.png "CoinAPI library item")
+   <br><br>
 
-Let’s create a free Twilio account:
+3. Click **Preview Code**, and on the page that appears, click **Edit Code**.
+    * Optionally, you can also jump straight to **Setup & Deploy** but we want to take a quick look at what the code is doing first, and perhaps change a few things.
```

Comment:
> Optionally, you can also jump straight to **Setup & Deploy** but we want to take a quick look at what the code is doing first, and perhaps change a few things.

Possibly better as something like:

Optionally, you could click **Setup & deploy**, to deploy the library item immediately. However, in this case, clicking `Preview code` allows you to view the implementation, and modify the code if necessary.